values.yaml

```diff
@@ -346,3 +361,15 @@ affinity: {}
 ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
 hooks:
   affinity: {}
+
+## -------------------------------------------------------------------------------
+## KEDA RabbitMQ Configuration
+## -------------------------------------------------------------------------------
+
+keda:
```

Comment:
can we add an enabled here (default false), so we can use that to add the keda pieces.